component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/GenericInputValidator.java

```diff
@@ -67,6 +67,7 @@
  * inputs will be validated.
  */
 public class GenericInputValidator {
+
     private static final org.wso2.am.analytics.publisher.reporter.GenericInputValidator INSTANCE =
             new org.wso2.am.analytics.publisher.reporter.GenericInputValidator();
     private static final Map<String, Class> defaultResponseEventSchema = Stream.of(
```

Comment:
Are we adding these new attributes only for moseif? or same published for choreo as well?